python/hail/methods/statgen.py

```diff
@@ -1697,6 +1697,9 @@ def genetic_relatedness_matrix(call_expr) -> BlockMatrix:
     Note that variants for which the alternate allele frequency is zero or one are not
     normalizable, and therefore removed prior to calculating the GRM.
 
+    This method drops variants with :math:`p_j = 0` or math:`p_j = 1` before
```

Comment:
This is a just a repeat of the note directly above, right? I would include either this note or the one above but not both.